.gitignore

```diff
@@ -16,3 +16,4 @@ data
 .project
 .settings/
 bin/
+plugins/
```

Comment:
Pre-emptive, or is this something that pf4j is doing?

Reply:
When running locally with a remote update repository configured plugins are downloaded to the projects root/plugins directory (created if it doesn't exist).